spec/model/posts_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before :each do
+    @user = User.new(id: 1, Name: 'Kenny', Bio: 'I am the greatest being alive', PostCounter: 1,
+                     Photo: 'https://google.com/googleIcon.png')
+
+    @post = Post.new(title: 'Love', text: 'My everthing', author_id: 1, CommentCounter: 1, LikesCounter: 0)
+  end
+
+  it 'title must not be blank' do
+    @post.title = nil
+    expect(@post).to_not be_valid
+  end
```

Comment:
- [ ] According to the project requirement, 👇🏻 Any custom methods in the models should have a corresponding unit test. You're missing adding unit test of the:
1.  `update_post_counter and five_recent_comments` methods of your post model.
2. `update_comment_counter` method of your comment model and,
3. `update_likes_counter` method of your like model.
### please address this ✍🏻 

![im2](https://user-images.githubusercontent.com/79058364/215210008-f647c979-9f49-40cc-a73e-e5d3c29cfbb1.png)